modules/get-started/pages/architecture.adoc

```diff
@@ -47,7 +47,9 @@ If a follower triggers an election, but the incumbent leader subsequently spring
 
 Redpanda stores metadata update commands (such as creating and deleting topics or users) in a system partition called the controller partition. A new snapshot is created after each controller command is added, or, with rapid updates, after a set period of time (default is 60 seconds). Controller snapshots save the current cluster metadata state to disk, so startup is fast. For example, with a partition that has moved several times, a snapshot can restore the latest state without replaying every move command.
 
-Each broker has a snapshot file stored in the controller log directory, such as `/var/lib/redpanda/data/redpanda/controller/0_0/snapshot`. The controller partition is replicated by a Raft group that includes all cluster brokers, and the controller snapshot is the Raft snapshot for this group. Snapshots are hydrated when a broker joins the cluster or restarts. Controller snapshots are enabled by default only in new clusters. To enable controller snapshots in existing or upgraded clusters, contact https://support.redpanda.com/hc/en-us[Redpanda Support^].
+Each broker has a snapshot file stored in the controller log directory, such as `/var/lib/redpanda/data/redpanda/controller/0_0/snapshot`. The controller partition is replicated by a Raft group that includes all cluster brokers, and the controller snapshot is the Raft snapshot for this group. Snapshots are hydrated when a broker joins the cluster or restarts. 
+
+The controller snapshots feature was released in Redpanda version 23.2. When older clusters are upgraded to a version supporting controller snapshots, this feature is disabled by default. To enable controller snapshots on these clusters, contact https://support.redpanda.com/hc/en-us[Redpanda Support^]. After controller snapshots is enabled, it will persist across cluster restarts and future upgrades. New clusters created subsequently automatically enable controller snapshots.
```

Comment:
Actually starting from 23.3 this is no longer true - the feature is enabled for all clusters, both new and upgraded. To summarize:
* 23.1 -> 23.2 upgrade: disabled
* new 23.2 cluster: enabled
* 23.3 cluster (both new and upgraded): enabled

Reply:
So this guidance only applies in version 23.2.

Reply:
Given that we do not go back and update older versions, seems like this is all a moot point. Updating per new feedback.